### DISCUSSION TITLE
Transitioning Keymodes in the Middle of the Map/Song

### DISCUSSION THREAD

**Discussion by Potaterr** [ppy/osu#25479] [Ideas] [answered]
I was wondering if it is possible to implicate for future updates in lazer, where in the middle of the map you could transition from a key mode to another in the middle of the map, like going from 4k to 7k to make a beat drop more of a difficult sensation. I was just wondering if this was possible and might give a new idea, idk. Sorry about probably wasting your time. 
PS: Im sorry If someone has already posted something about this, I searched for keywords n looked around but i didnt find anything similar. Also im extremely new to this website so I dont know anything.

**Comment by peppy** [comment, marked as answer]
It's not currently possible, but given community support anything could be possible.
